config/blueprint.php

```diff
@@ -102,4 +102,18 @@
 
     'use_guarded' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Use Route Tuples
+    |--------------------------------------------------------------------------
+    |
+    | By default, Blueprint follows the Laravel convention of the controller
+    | namespace matches the namespace set within the RouteServiceProvider.
+    | However, you may configure Blueprint to generate routes using a
+    | "tuple syntax" in cases where you may not use this property
+    | or wish to improve static analysis.
+    |
+    */
+    'use_route_tuples' => false,
```

Comment:
```suggestion
    'use_fqn_routes' => \Illuminate\Foundation\Application::VERSION[0] !== '7',
```

Reply:
That's a bit fancy. I worry that will trip devs up when they upgrade their app. Consider that fact that just because newly created Laravel 8 apps will not have it set, all existing apps will.

Reply:
Hm, true.